packages/visual-editor/src/internal/puck/components/LayoutHeader.tsx

```diff
@@ -47,20 +49,12 @@ export const LayoutHeader = (props: LayoutHeaderProps) => {
 
   const [approvalModalOpen, setApprovalModalOpen] =
     React.useState<boolean>(false);
-
-  const {
-    appState,
-    history: {
-      back,
-      forward,
-      histories,
-      index,
-      hasPast,
-      hasFuture,
-      setHistories,
-    },
-  } = usePuck();
   const { i18n } = usePlatformTranslation();
+  const getPuck = useGetPuck();
```

Comment:
Is createUsePuck a superset of useGetPuck (used on line 23)? It sounded like it from your commits desc, except the former also rerenders.